src/index.js

```diff
@@ -73,10 +73,26 @@ function optimizeAndCompileSchema({schema, profileDocuments, compiler}) {
 }
 
 function prependFragments(source, id) {
+  return recursivelySourceFragmentFiles(source, id).then((fragments) => {
+    return source.concat(fragments.map((fragment) => fragment.body).join('\n'));
+  });
+}
+
+function recursivelySourceFragmentFiles(source, id) {
```

Comment:
Minor thing, but can this argument order match up? Just a little weird to see `source, id` then it calls `id, source`.

Reply:
I'm not sure which direction to go here. The rollup convention is `source, id`. In a bunch of spots in the library this thing depends on the convention is `path, body`, which translates to `id, source`. No matter which way I go, it's inconsistent somewhere. What I did here was match the rollup convention. Thoughts?